test/e2e/set/scyllacluster/scyllacluster_evictions.go

```diff
@@ -37,7 +36,10 @@ var _ = g.Describe("ScyllaCluster evictions", func() {
 		o.Expect(err).NotTo(o.HaveOccurred())
 
 		verifyScyllaCluster(ctx, f.KubeClient(), sc)
-		hosts := getScyllaHostsAndWaitForFullQuorum(ctx, f.KubeClient().CoreV1(), sc)
+		waitForFullQuorum(ctx, f.KubeClient().CoreV1(), sc)
+
+		hosts, err := utils.GetBroadcastRPCAddresses(ctx, f.KubeClient().CoreV1(), sc)
```

Comment:
I wonder (in general) whether we shouldn't keep a client open and test it works / updates the hostid during the rolling restart). real workloads will refresh hosts from scratch only on client restarts

Reply:
I agree, and I removed the code reestablishing the session (`di.SetClientEndpoints`) wherever possible, so basically any test running more than one node. This tests only inserts and verifies data once, but many tests have extra checks `verifyCQLData(ctx, di)` without resetting the connection through their duration.